src/modules/login/login.controller.js

```diff
@@ -0,0 +1,6 @@
+// Login controller
+'use strict';
+
+login.controller('loginCtrl', function () {
```

Comment:
Yes, but we are waiting for the issue to arise in order to learn. When it happens we will tackle dependency injection.